mantidproject/mantid
# Change IMDDimension::getName() to name.

Description of work.

This deprecates the python member function IMDDimension::getName(), which isn't terribly pythonic. The read-only data member IMDDimension::name has been added and should be used instead.

**To test:**



Review changes to the python interface for IMDDimension. Most other changes are just renaming that should be caught by the build servers.

There is no GitHub issue associated with this pull request.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
